tests/dynamicFormSelect.test.ts

```diff
@@ -65,45 +70,23 @@ describe('Dynamic form select component', function () {
                 formControl
             }
         });
-        const options = rendered.findAll("option");
+        const options = (rendered.vm as any).formOptions;
         expect(options.length).toBe(2);
-        expect((options[0].element as HTMLOptionElement).value).toBe("opt1");
-        expect(options[0].text()).toBe("option 1");
+        expect(options[0].id).toBe("opt1");
+        expect(options[0].label).toBe("option 1");
 
         expect(rendered.emitted("update:formControl")).toBeTruthy();
         expect(rendered.emitted("update:formControl")![0][0]).toEqual({...formControl, value: "opt1"});
     });
 
-    it("is required if formControl.required is true", () => {
-        const rendered = mount(DynamicFormSelect, {
-            props: {
-                formControl: {...fakeSelect, required: true}
-            }
-        });
-
-        const select = rendered.find("select");
-        expect((select.element as HTMLSelectElement).required).toBe(true);
-    });
-
-    it("is not required if formControl.required is false", () => {
-        const rendered = mount(DynamicFormSelect, {
-            props: {
-                formControl: {...fakeSelect, required: false}
-            }
-        });
-
-        const select = rendered.find("select");
-        expect((select.element as HTMLSelectElement).required).toBe(false);
-    });
-
```

Comment:
Why have these been removed? Is that not part of the interface any more?

Reply:
so we just didnt have the required prop set in Hint version of the refactor, but i looked at the component and it did have a required prop in the vue2 version so assuming it does have this in the vue 3 version, i can add it back in!